### DISCUSSION TITLE
what am I missing?

### DISCUSSION THREAD

**Discussion by Dushku** [hippie68/ftpdump#21] [Q&A] [answered]
I have taken the ftp.bin and put it on my PS4 at /data/payloads. I have used Payload Guest to run it and taken note of the IP and port.

I edited ftpdump to have my PS4's IP and port set.

I have installed WSL on my 64-bit Win10.

When I open up cmd.exe in Windows and type "ftpdump CUSA10938", it tells me "'ftpdump' is not recognized as an internal or external command, operable program or batch file."

What am I missing please? Thank you for your time.

**Comment by hippie68** [comment, marked as answer]
When using cmd.exe, you have to navigate to the folder that contains ftpdump.bat:

    cd Path\To\Folder

If you just want to dump the currently running app, you don't need to specify any ID.
